### PR TITLE
COMMERCE-5586 Fix queries by channel

### PR DESCRIPTION
**Pull request by riccardo-alberti**

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-price-list-service/src/main/resources/META-INF/custom-sql/default.xml [hunk 1]
```diff
@@ -283,8 +283,16 @@
...
+			LEFT JOIN
+				CPLCommerceGroupAccountRel ON
+					CommercePriceList.commercePriceListId = CPLCommerceGroupAccountRel.commercePriceListId
+			LEFT JOIN
+				CommercePriceListAccountRel ON
+					CommercePriceList.commercePriceListId = CommercePriceListAccountRel.commercePriceListId
 			WHERE
 				(CommercePriceListChannelRel.commerceChannelId = ?) AND
+				(CommercePriceListAccountRel.commercePriceListAccountRelId IS NULL) AND
+				(CPLCommerceGroupAccountRel.cPLCommerceAccountGroupRelId IS NULL) AND
```
Comment (dsitu) [inline modules/apps/commerce/commerce-price-list-service/src/main/resources/META-INF/custom-sql/default.xml:295]:
```suggestion
				(CPLCommerceGroupAccountRel.CPLCommerceAccountGroupRelId IS NULL) AND
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**